lib/prefs/settings.js

```diff
@@ -101,6 +101,7 @@ export class SettingsPage extends PreferencesPage {
           type: "s",
           bind: "dnd-center-layout",
           items: [
+            { id: "none", name: _("None (swap windows)") },
```

Comment:
Hi @alesya-h - would it be OK to use `Swap`  _versus_ `None (Swap Windows)`? Same with the expressions if (layout == none) update to (layout == swap).

Reply:
Sure, updated